kuma/wiki/tests/test_views.py

```diff
@@ -1720,8 +1714,7 @@ def test_edit_spam_ajax(self, mock_requests, mock_akismet_method, translate_loca
         )
 
         spam_message = render_to_string('wiki/includes/spam_error.html')
-        ok_(spam_message in json.loads(resp.content)['error_message'],
-            "Spam message should appear")
+        assert spam_message in json.loads(resp.content)['error_message'], "Spam message should appear"
```

Comment:
Again, since the line exceeds the PEP8 line-length limit and the assert error message doesn't really add any value, I would suggest dropping the error message:
```python
assert spam_message in json.loads(resp.content)['error_message']
```